### PR TITLE
Feature/update user profile

### LINKED ISSUE #50: Update shipping address
**Is your feature request related to a problem? Please describe.**
User can't update their shipping address.

**Describe the solution you'd like**
As a user, I'd like to be able to change my shipping address so that I can keep it updated.

**Additional context**

- [x] Add button in user profile to edit shipping address
- [x] Add input field(s) for new shipping address to be entered
- [x] Utilize helper functions (?) and error handling on shipping address field
- [x] What is the format of the shipping address in the API?
- [ ] Pass new shipping address to API to be saved on user object
- [x] Show new shipping address on user profile

### PR DESCRIPTION
**Pull request by elsafluss**
## *Purpose:*

- [x] Adds a new feature
- [x] Improves an existing feature
- [x] Fixes a bug
- [x] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
User's information is pushed to localStorage and API so that it can persist.

## *What are the relevant tickets?*
Closes #50 #51 #48 #47 #49 

## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: code/web/src/modules/user/Profile.js [hunk 1]
```diff
@@ -75,22 +75,45 @@ class Profile extends PureComponent {
   }
 
   // Submit controlled form data
-  handleSubmit = () => {
+  handleSubmit = (event) => {
+    event.preventDefault()
+    this.props.updateUser(this.state.currentUser)
+    window.localStorage.setItem('user', JSON.stringify(this.state.currentUser))
```
Comment (mattdeann) [inline code/web/src/modules/user/Profile.js:81]: Nice fix for this!

File: code/web/src/modules/user/api/actions.js [hunk 2]
```diff
@@ -90,6 +90,7 @@ export function register(userDetails) {
 
 // Update user data
 export function updateUser(userDetails) {
+  console.log(userDetails)
```
Comment (alia-peterson) [inline code/web/src/modules/user/api/actions.js:93]: probably need to remove this before merging!

**Review by alia-peterson — APPROVED** [review]
- [ ] Changes generate no new warnings or errors
- [ ] Branch was tested locally without failures
- [ ] All tests are passing as expected

File: code/web/src/modules/user/api/actions.js (hunk 2 above)
Comment (mattdeann) [inline code/web/src/modules/user/api/actions.js:93]: Take this out later!

**Review by mattdeann — APPROVED** [review]

**Pull request merged**